src/seedu/addressbook/data/AddressBook.java

```diff
@@ -107,6 +107,13 @@ public void clear() {
         allTags.clear();
     }
 
+    /**
+     * Stably sorts all persons in the address book by name.
```

Comment:
Same as previously stated, perhaps look to be more precise in stating the functionality of the method. Is it sorted is lexicographical order?